resources/js/Pages/Results.vue

```diff
@@ -267,4 +310,25 @@ h2 {
     margin-top: $dimension-layout-xsmall;
 }
 
+p.list-intro {
+    margin-bottom: 0
+}
+
+#description-section {
+    position:relative;
+
+    .description-header {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
```

Comment:
In this case, we should also add `justify-content: space-between` which will ensure that each child of the header will take the most extreme side.